pkg/tfvars/ovirt/ovirt.go

```diff
@@ -0,0 +1,36 @@
+// Package ovirt contains ovirt-specific Terraform-variable logic.
+package ovirt
+
+import (
+	"encoding/json"
+)
+
+type config struct {
+	URL        string `json:"ovirt_url"`
+	Username   string `json:"ovirt_username"`
+	Password   string `json:"ovirt_password"`
+	Cafile     string `json:"ovirt_cafile,omitempty"`
```

Comment:
I don't think this is used... there is not a corresponding variable in variables-ovirt.tf